### PR TITLE
source files were added

### PR DESCRIPTION
**Pull request by volodymyrtaras**

### CODE REVIEW

File: src/main/java/dao/PersonServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+/*************************************
+ * This class implements PersonService
+ * and realise its methods
+ *************************************/
+
+@Service("personService")
+public class PersonServiceImpl implements PersonService {
+    @Autowired
+    PersonDao personDao;
```
Comment (Taniann) [inline src/main/java/dao/PersonServiceImpl.java]: better add access modificator

File: src/test/java/PersonDaoImplTest.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    @Test
+    public void editPersonTest() {
+        Volodymyr.setFirstName("Volodymyr").setLastName("Taras").setAge(20);
+        personService.editPerson(Volodymyr, 50);
+        Assert.assertEquals(personService.find(50).getFirstName(),"Volodymyr");
+    }
+
+    @Test
+    public void deletePersonTest() {
+        personService.deletePerson(49);
```
Comment (Taniann) [inline src/test/java/PersonDaoImplTest.java]: you arent asserting here anything

File: src/main/java/dao/PersonDao.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ *********************************/
+
+public interface PersonDao {
+    void addPerson(Person person);
+
+    void editPerson(Person person, int personId);
+
+    void deletePerson(int personId);
+
+    Person find(int personId);
```
[the remark this answers is not in the document]
Comment (volodymyrtaras) [reply src/main/java/dao/PersonDao.java]: @Taniann I have done it, but Aleksandr told us that in real life sometimes it is better to get null

[the remark this answers is not in the document]
Comment (volodymyrtaras) [reply src/main/java/dao/PersonDao.java]: @Taniann 

https://tuhrig.de/anti-pattern-dont-use-optionals-for-data-repositories/

[the remark this answers is not in the document]
Comment (volodymyrtaras) [reply src/main/java/dao/PersonDao.java]: @Taniann and it is realy confused for me - testing Optional object)

**Comment by volodymyrtaras** [comment]
> resolve comments and add controller

Aleksandr said on the lesson, that controller isn`t necessary

**Comment by volodymyrtaras** [comment]
@fixed.

**Comment by volodymyrtaras** [comment]
@fixed.

**Pull request merged**